src/backend/opencl/platform.cpp

```diff
@@ -673,22 +672,26 @@ void DeviceManager::markDeviceForInterop(const int device, const fg::Window* wHa
 #if defined(_WIN32) || defined(_MSC_VER)
                 CL_WGL_HDC_KHR, (cl_context_properties)wHandle->display(),
 #else
-                CL_GLX_DISPLAY_KHR, (cl_context_properties)wHandle->display(),
+                //CL_GLX_DISPLAY_KHR, (cl_context_properties)wHandle->display(),
 #endif
                 CL_CONTEXT_PLATFORM, (cl_context_properties)plat(),
```

Comment:
You can't dot hat when the devices are from different platforms. I am testing this on my personal machine which has an AMD GPU as well as an NVIDIA GPU.